license.rst

```diff
@@ -0,0 +1,72 @@
+==========================
+END USER LICENSE AGREEMENT
+==========================
+
+for Anaconda, Miniconda, and Conda
```

Comment:
Where does this appear in the program?

On Monday, May 18, 2015, Aaron Meurer <notifications@github.com> wrote:

> In license.rst
> <https://github.com/conda/conda-docs/pull/55#discussion_r30553078>:
>
> > @@ -0,0 +1,72 @@
> > +==========================
> > +END USER LICENSE AGREEMENT
> > +==========================
> > +
> > +for Anaconda, Miniconda, and Conda
>
> The Anaconda license is different from the conda license. The conda
> license is the BSD license listed here
> <https://github.com/conda/conda/blob/master/LICENSE.txt>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/conda/conda-docs/pull/55/files#r30553078>.
>